frontend/src/metabase/visualizations/visualizations/SmartScalar.jsx

```diff
@@ -145,17 +195,51 @@ export default class Smart extends Component {
       ? color("error")
       : color("success");
 
-    const changeDisplay = (
-      <span style={{ fontWeight: 900 }}>
-        {formatNumber(Math.abs(lastChange), { number_style: "percent" })}
-      </span>
-    );
-    const separator = (
-      <PreviousValueSeparator gridSize={gridSize}>•</PreviousValueSeparator>
-    );
-    const granularityDisplay = (
-      <span style={{ marginLeft: 5 }}>{jt`last ${granularity}`}</span>
+    const titleLinesCount = getTitleLinesCount(height);
+    const availableWidth = width - 2 * SPACING;
+    const availableChangeWidth = availableWidth - ICON_SIZE - ICON_MARGIN_RIGHT;
+
+    const availableValueHeight =
+      height -
+      titleLinesCount * SCALAR_TITLE_LINE_HEIGHT -
+      MIN_PREVIOUS_VALUE_SIZE -
+      4 * SPACING;
+
+    const tooltipSeparator = <Separator>•</Separator>;
+
+    const changeDisplay = formatChangeAutoPrecision(lastChange, {
+      fontFamily,
+      fontWeight: 900,
+      width: availableChangeWidth,
+    });
+
+    const changeWidth = measureText(changeDisplay, {
+      size: "1rem",
+      family: fontFamily,
+      weight: 900,
+    }).width;
+
+    const granularityDisplay = jt`last ${granularity}`;
+    const previousValueDisplay = formatValue(
+      previousValue,
+      settings.column(column),
     );
+    const disabledSeparator = ""; // avoid creating new translation key
+    const previousValueContent = jt`${disabledSeparator} was ${previousValueDisplay} ${granularityDisplay}`;
```

Comment:
I'm not sure why separator was originally included in the translation.

Please let me know if I should introduce a new translation key.